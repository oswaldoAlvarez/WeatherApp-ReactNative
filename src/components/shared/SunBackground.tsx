import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

export const SunBackground = () => {
  return <View style={styles.container} />;
};

const styles = StyleSheet.create({
  container: {
    position: 'absolute',
    backgroundColor: '#FFCC45',
    top: -250,
    right: -300,
    width: 500,
    height: 500,
    borderRadius: 500,
    transform: [{ rotate: '-50deg' }],
  },
});
