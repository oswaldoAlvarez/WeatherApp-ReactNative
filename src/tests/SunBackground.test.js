import React from 'react';
import renderer from 'react-test-renderer';
import { SunBackground } from '../components/shared/SunBackground';

test('Probando funciones del SunBackground', () => {
  const tree = renderer.create(<SunBackground />).toJSON();
  expect(tree).toMatchSnapshot();
});
