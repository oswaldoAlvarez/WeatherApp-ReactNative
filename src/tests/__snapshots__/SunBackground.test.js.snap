// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando funciones del SunBackground 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFCC45",
      "borderRadius": 500,
      "height": 500,
      "position": "absolute",
      "right": -300,
      "top": -250,
      "transform": Array [
        Object {
          "rotate": "-50deg",
        },
      ],
      "width": 500,
    }
  }
/>
`;
